### PR TITLE
docs: add IBC setup using DIVE CLI to readme

### PR DESCRIPTION
**Pull request by shreyasbhat0**
### Commit Message

```bash
docs: add IBC setup using DIVE CLI to readme
```

see the [guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages) for commit messages.

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have documented my code in accordance with the [documentation guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#documentation)
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the unit tests
- [ ] I only have one commit (if not, squash them into one commit).
- [x] I have a descriptive commit message that adheres to the [commit message guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages)
- [ ] I have added version bump label on PR.

> Please review the [CONTRIBUTING.md](/CONTRIBUTING.md) file for detailed contributing guidelines.

### CODE REVIEW

**Review by ibrizsabin — DISMISSED** [review]
Looks good to me. Thanks shreyas , nice work.

File: README.md [hunk 1]
```diff
@@ -103,6 +96,10 @@ Make sure you have following installed on your machine to build the contracts or
 | [/test](./test)                                   | Test Framework and Test Suite including e2e test and functional test           |
 | [/utils](./utils)                                 | Utilities used for build, setup, CI/CD                                         |
 
+## Local Setup And Runnning Tests Using DIVE CLI
+
+For Setting up nodes , deploy contracts and end-to-end testing follow setps provided [here](./docs/ibc-setup-using-dive-cli.md)
```
Comment (izyak) [inline README.md:101]: steps, typo

File: docs/ibc-setup-using-dive-cli.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  ```
+
+  - Spins icon and neutron node
+  - Opens Btp Network for BTP Blocks
+  - Deploys the Contracts necessary for the IBC connection
+  - Starts the relay
+
+## End-to-End Testing
+
+To run end-to-end testing script please follow from the second setp mentioned [here](https://github.com/HugoByte/DIVE/blob/main/test/README.md#end-to-end-icon---archway-demo)
```
Comment (izyak) [inline docs/ibc-setup-using-dive-cli.md:37]: second `step` mentioned, typo here

File: README.md (hunk 1 above)
Comment (shreyasbhat0) [reply README.md]: Fixed

File: docs/ibc-setup-using-dive-cli.md (hunk 2 above)
Comment (shreyasbhat0) [reply docs/ibc-setup-using-dive-cli.md]: Fixed

**Review by AntonAndell — APPROVED** [review]
LGTM.
Was easy enought to run :+1:

**Pull request merged**